trompace/mutations/controlaction.py

```diff
@@ -90,3 +90,50 @@ def mutation_modify_controlaction(controlaction_id: str, actionstatus: ActionSta
         args["error"] = error
 
     return mutation_create(args, UPDATE_CONTROLACTION)
+
+
+def mutation_add_controlaction_propertyvaluespecification(controlaction_id: str, propertyvaluespecification_id: str):
```

Comment:
These mutations that join two items together can be difficult to conceptualise, and because of that I decided to be consistent with the naming of the function given the mutation name.

The method name and arguments should directly match the name of the mutation, so in this case
MergePropertyValueSpecificationPotentialAction:
   - from: PropertyValueSpecification
   - to: ActionInterface (field name: PotentialAction, type Action)

method name: `mutation_add_propertyvaluespecification_potentialaction`
arguments: propertyvaluespecification_id and action_id

Always name arguments so that the match the order of the items in the mutation, so propertyvaluespecification_id first.

See
https://github.com/trompamusic/trompa-ce-client/blob/95cdc54979c53d2d3c567ff97ea7129f07ffe69c/trompace/mutations/musiccomposition.py#L168-L176
 for another example. Note that here I named the second argument after the _type_ of the field, not the name of the field.